client/eventsSDL/InputSourceKeyboard.cpp

```diff
@@ -30,13 +31,21 @@ InputSourceKeyboard::InputSourceKeyboard()
 
 void InputSourceKeyboard::handleEventKeyDown(const SDL_KeyboardEvent & key)
 {
-	if(key.repeat != 0)
-		return; // ignore periodic event resends
-
 	if (SDL_IsTextInputActive() == SDL_TRUE)
 	{
+		if(key.keysym.sym == SDLK_v) {
```

Comment:
Shouldn't you also check for Ctrl?
There is `isKeyboardCtrlDown` in this class that you might want to use

Reply:
You're absolutely right. Lost at debugging. 😅